### ISSUE TITLE
[Bug]: Lots of WARNs in Log: "[ApiRouter] Library item not found for users progress" ...

### ISSUE THREAD

**Issue by Lebo77** [advplyr/audiobookshelf#921]
### Describe the issue

Not a functionality issue, but my log is full of warnings like this:

```
`2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_5fgvs5w2hj0gvkniwl-ep_zw0k9m3n71r2j0x18m

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_v8yk3gftzhmrsrkqt3-ep_7hz65t5dyk5t3jzfop

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_v8yk3gftzhmrsrkqt3-ep_prrhkd140fivnu7yaz

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_v8yk3gftzhmrsrkqt3-ep_e9c5kihlhfiffqdgdl

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_qkuoc3jlscgbepvymj-ep_73bp0zzytirdd7ba3a

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_8l2e7qqwhmbdlc5quv-ep_0werhpi7zmxp3nww58

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_8l2e7qqwhmbdlc5quv-ep_uvu5o1r0k65qtr11pk

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_8l2e7qqwhmbdlc5quv-ep_thjit8u15xg6e20lc8

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_8l2e7qqwhmbdlc5quv-ep_od0yloksyyong0p2mv

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_h0dk8y339ul3gh2y56-ep_yxz9uamfirwn2xl0mh

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_3asmomod5mnzq1693g-ep_bvshfxaimbhmy3lmei

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_mdowmsov3iu4vklhk3-ep_8kk8c48uprjwd95gxs

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_mdowmsov3iu4vklhk3-ep_fmuwc3afdukzrvzg4p

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_3asmomod5mnzq1693g-ep_ua2lpi6aw08in09kkg

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_mc6woq913ool8ws29o-ep_l9o45g6ktoisoqstq4

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_s9kftj6zvfdu3nscm4-ep_dcpvw9lzwu3hhkuok7

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_mc6woq913ool8ws29o-ep_kc65uc2ci5j246icq7

2022-08-20 21:26:11

WARN

[ApiRouter] Library item not found for users progress li_mc6woq913ool8ws29o-ep_wplxx65xzkr2wsj5g9`
```

### Steps to reproduce the issue

1. Unknown. I just saw them in the log. I am guessing you need to have listened to something, then deleted the underlying file.

### Audiobookshelf version

v 2.1.3

### How are you running audiobookshelf?

Docker

**Comment by skyzuma** [comment]
i also have a few line of this but idk why

>WARN: [ApiRouter] Library item not found for users progress li_gn4ymda1xv72abbfnq-ep_3ctcbfr4c5jyuajn2x
WARN: [ApiRouter] Library item not found for users progress li_gn4ymda1xv72abbfnq-ep_7alovemhdbajvkvenp
WARN: [ApiRouter] Library item not found for users progress li_gn4ymda1xv72abbfnq-ep_528ol0zr9vqhfi28h4
WARN: [ApiRouter] Library item not found for users progress local_cHJpbWFyeTpBQm9va3MvQWxsZSBkaWVzZSBXZWx0ZW4=

**Comment by advplyr** [comment]
These logs are from going to the users page and having media progress stored on the user that doesn't have an associated library item anymore.
We still need to setup periodic purging of old data.

**Comment by advplyr** [comment]
Fixed in [v2.2.0](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.0)

There is also a new button to purge old media progress items on the users config page, but this shouldn't be needed normally.
